R/run_stress_test.R

```diff
@@ -366,3 +346,52 @@ iteration_sequence <- function(args_list) {
     return(seq_along(args_list[[iter_var]]))
   }
 }
+
+st_read <- function(input_path_project_specific, asset_type) {
+  path <- pacta_results_path(input_path_project_specific, asset_type)
+
+  out <- list(
+    pacta_results = read_pacta_results(path)
+  )
+
+  return(out)
+}
```

Comment:
I plan to extend this function in other PRs. I don't do it now to keep the diff small. Eventually it will read each kind of data.